src/thunderbird_accounts/mail/views.py

```diff
@@ -5,9 +5,14 @@
 
 # Create your views here.
 
+
 def home(request: HttpRequest):
-    return TemplateResponse(request,'mail/index.html', {})
+    return TemplateResponse(request, 'mail/index.html', {})
 
 
 def self_serve(request: HttpRequest):
-    return TemplateResponse(request,'mail/self-serve.html', {})
+    return TemplateResponse(request, 'mail/self-serve.html', {})
+
+
+def waitlist(request: HttpRequest):
+    return TemplateResponse(request, 'mail/waitlist.html', {})
```

Comment:
```suggestion
    return TemplateResponse(request, 'mail/wait-list.html', {})
```
(and do a rename on that template)